src/frontend/src/views/Resources/index.tsx

```diff
@@ -0,0 +1,228 @@
+              </span>,
+              <span key={5}>
+                <NewTabLink href="https://www.youtube.com/watch?v=GJMTNwDKibI">
+                  Week 6
+                </NewTabLink>{" "}
+                - This week we talked all about data types and data
+                organization. We discussed how using data models helps to
+                organize genomic surveillance metadata and specifically
+                mentioned the PHA4GE data specification for SARS-CoV-2. And then
+                Dan Lu walked us through different genomic data structures, what
+                those different structures are useful for, and which
+                repositories each can be submitted to. She finished the talk off
+                with a discussion of handling GISAID rejections and looking at
+                calls in BAM files.
+              </span>,
+              <span key={6}>
+                <NewTabLink href="https://www.youtube.com/watch?v=HVorizRS4wk">
+                  Week 7
+                </NewTabLink>{" "}
+                - In todays seminar we talked about phylogenetic nomenclature
+                systems (why we use them and types of systems). We finished off
+                with a more detailed discussion of how the Pango nomenclature
+                system for SARS-CoV-2.
+              </span>,
+              <span key={7}>
+                <NewTabLink href="https://www.youtube.com/watch?v=ycM50nC5wXk">
+                  Week 8
+                </NewTabLink>{" "}
+                - This week we tackled the tricky subject of sampling design.
+                While we don&apos;t (yet) have formal frameworks for estimating
+                sample size in genomic epidemiology, today we discussed study
+                design, and specifically how to think about sample selection for
+                different types of genomic epidemiological questions/studies.
+              </span>,
+              <span key={8}>
+                <NewTabLink href="https://www.youtube.com/watch?v=a8bp0iHCttA">
+                  Week 9
+                </NewTabLink>{" "}
+                - In this week&apos;s seminar I introduced
+                &quot;phylodynamic&quot; analysis - a particular area of genomic
+                epidemiology where we infer changes in pathogen population size
+                from the shapes of coalescent phylogenetic trees.
+              </span>,
+              <span key={9}>
+                <NewTabLink href="https://www.youtube.com/watch?v=cLLV2VpgXCU">
+                  Week 10
+                </NewTabLink>{" "}
+                - This week I gave a demo of Aspen, the new piece of software
+                that CZI is developing for managing genomic data, that
+                eventually will allow folks to build Nextstrain trees
+                independently with a GUI interface. Since that capability
+                isn&apos;t active quite yet, I also discussed phylogenetic
```

Comment:
Maybe specifically point out the "Aspen Tutorial!" section at the bottom of the list since it's showing those capabilities off? Or just have a direct link to the video in the editorial aside that points to the same place as "Aspen Tutorial!" does?